.github/pull_request_template.md

```diff
@@ -28,8 +28,11 @@ Pick one of the following and delete the others:
 - Misc (other changes)
 
 ###  How is the PR tested?
-
-Please describe the tests that you ran to verify your changes. Have you added unit or integration tests, or run manual tests? What additional tests would give you greater confidence in this change?
+Pick some of the following tests which be tested
```

Comment:
The description is a little weird, but I have no idea with it. What's your opinion? @dcalvin

Reply:
Already suggested a change above.